crates/config/src/unresolved_config.rs

```diff
@@ -31,6 +31,7 @@ const DEFAULT_UTILITY_THREAD_ID: u32 = 1;
 #[serde(rename_all = "kebab-case")]
 pub struct UnresolvedCCPConfig {
     pub http_server: UnresolvedHTTPServer,
+    pub prometheus_endpoint: UnresolvedPrometheusEndpoint,
```

Comment:
I suggest making prometheus metrics optional, so we can run CCP with disabled PrometheusClient.

Reply:
Makes sense.